apk-generator/v2/app/generator/generator.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, config, via_api=False, identifier=None, task_handle=None):
         self.api_link = ''
         self.apk_path = ''
         self.via_api = via_api
+	self.build_type = build_type
```

Comment:
also when passing a variable from python to .sh file it would be accessed like this $buildType or ${buildType} ?